usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -81,6 +87,8 @@
     "sub_place_of_perform_state_code": {"delta": "STRING", "postgres": "TEXT"},
     "sub_place_of_perform_state_name": {"delta": "STRING", "postgres": "TEXT"},
     "sub_place_of_performance_zip": {"delta": "STRING", "postgres": "TEXT"},
+    "sub_place_of_performance_county_code": {"delta": "STRING", "postgres": "TEXT"},
```

Comment:
Our code base hard codes the pattern of these these legal entity and performance location columns. It requires that they follow this pattern, "sub_place_of_perform_" and "sub_legal_entity_". This column name goes against the pattern and I anticipate it will cause issues. For example, see the file `/workspaces/usaspending-api/usaspending_api/search/v2/views/spending_by_geography.py` method `post` on line 70.

Reply:
If that's the case, the downloads would need to update too right? Since they are already existing and pointing to the old names? Wondering why this would be a problem now and not already since they're already like this.

Reply:
Ah I see. The columns we were trying to update had that subtle name change. Fixing.

Reply:
Ah I see. The columns we were trying to update had that subtle name change. Good catch.

Reply:
With the new columns, I kept them as `place_of_performance_county_*` as we currently don't use them in the code but can change them to the `place_of_perform_county_*` versions. Do you have a preference?

Reply:
>  we currently don't use them in the code

That's true, but if we do end up using them in the places that hard code this pattern it'll matter.

I think `place_of_perform_` is good